src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -1158,8 +1161,7 @@ public boolean isGameRule(String rule)
 	@Override
 	public WorldBorder getWorldBorder()
```

Comment:
Wrong return type and missing annotation

```suggestion
	public @NotNull WorldBorderMock getWorldBorder()
```